2023-03-16-dasharo-fidelisguard-z690-first-qubes-certified-desktop.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "The Dasharo FidelisGuard Z690 is the first Qubes-certified desktop computer!"
+categories: announcements
+image: /attachment/posts/dasharo-fidelisguard-z690_2.jpg
+---
+
+It is our pleasure to announce that the [Dasharo FidelisGuard Z690](https://3mdeb.com/shop/open-source-hardware/dasharo-fidelisguard-z690-qubes-os-certified/) has become the fourth [Qubes-certified computer](/doc/certified-hardware/) for Qubes 4.X and the **first** Qubes-certified desktop computer **ever**!
+
+## About the Dasharo FidelisGuard Z690
+
+[![Photo of MSI PRO Z690-A DDR4 motherboard](/attachment/posts/dasharo-fidelisguard-z690_1.jpg)](https://3mdeb.com/shop/open-source-hardware/dasharo-fidelisguard-z690-qubes-os-certified/)
+
+The [Dasharo FidelisGuard Z690](https://3mdeb.com/shop/open-source-hardware/dasharo-fidelisguard-z690-qubes-os-certified/) is a full desktop PC build that brings the [Dasharo](https://dasharo.com/) open-source firmware distribution to the MSI PRO Z690-A DDR4 motherboard with Qubes OS preinstalled. The full configuration includes:
+
+| Part         | Model Name                                                     |
+|------------- | -------------------------------------------------------------- |
+| CPU	         | Intel Core i5-12600K, 3.7GHz                                   |
+| Cooling	     | Noctua CPU NH-U12S Redux                                       |
+| RAM	         | Kingston Fury Beast, DDR4, 4x8GB (32 GB Total), 3600 MHz, CL17 |
+| Power Supply | Seasonic Focus PX 750W 80 Plus Platinum                        |
+| Storage      | SSD Intel 670p 512 GB M.2 2280 PCI-E x4 Gen3 NVMe              |
+| Enclosure	   | SilentiumPC Armis AR1                                          |
+
+[![Photo of Dasharo FidelisGuard Z690 with open case](/attachment/posts/dasharo-fidelisguard-z690_2.jpg)](https://3mdeb.com/shop/open-source-hardware/dasharo-fidelisguard-z690-qubes-os-certified/)
+
+This computer comes with a "Dasharo Supporters Entrance Subscription," which includes the following:
+
+- Full access to [Dasharo Tools Suite (DTS)](https://docs.dasharo.com/dasharo-tools-suite/overview/)
+- The latest Dasharo releases issued by the Dasharo Team
+- Special Dasharo updates for supporters
+- Dasharo Premier Support through an invite-only Matrix channel
```

Comment:
```suggestion
- Dasharo Primer Support through an invite-only Matrix channel
```

Reply:
Oh, I assumed "Primer" was a typo and that you meant "Premier." What does "Primer Support" mean?

Reply:
:thinking: @andrewdavidwong of course you are right we use this word wrong, it is interesting that nobody noticed that yet. Let's use Premier instead of incorrect Primer

Reply:
No worries! :slightly_smiling_face: